pages/common/gdu.md

```diff
@@ -0,0 +1,36 @@
+# gdu
+
+> Disk usage analyzer with console interface.
+> More information: <https://github.com/dundee/gdu>.
+
+- Show disk usage of the current directory:
+
+`gdu`
+
+- Show disk usage of given directory:
+
+`gdu {{path/to/directory}}`
+
+- Show disk usage of all mounted disks:
+
+`gdu --show-disks`
+
+- Show disk usage but ignore some paths:
+
+`gdu --ignore-dirs {{/sys,/proc}} /`
```

Comment:
If this page is under common, we sholudn't use Linux-specefic directories. Also, it would be easier if we remove the second argument.
```suggestion
- Show the disk usage of the current directory but ignore some sub-directories:

`gdu --ignore-dirs {{path/to/directory1,path/to/directory2,...}}`
```
You can of course change something if you like to.